### DISCUSSION TITLE
Cannot connect to OneDrive because of storage quota limit being reached.

### DISCUSSION THREAD

**Discussion by eros-fabbri** [abraunegg/onedrive#2937] [Q&A] [answered]
Hi everyone, i'm trying to use the onedrive client to download (with --download-only) but i get this error:

```
ERROR: OneDrive account currently has zero space available. Please free up some space online or purchase additional capacity.

ERROR: Microsoft OneDrive API returned an error with the following message:
  Error Message:    HTTP request returned status code 403 (Forbidden)
  Error Reason:     Access Denied
  Error Code:       accessDenied
  Error Timestamp:  2024-10-30T10:59:36
  API Request ID:   7e036736-8bf2-44e5-b2b4-f16c6d9e2516

Unable to query OneDrive API to obtain required account details
```
Is there any way to download everything? (not from the website)

**Comment by abraunegg** [comment, marked as answer]
@eros-fabbri 
This is not a zero space issue. Your account has lost access to your data. That is what a 403 error / Access Denied means.

Either:
* Your account is blocked
* Your account has been closed
* Your account password has changed

This is not something this client can fix.
